Formula/mingw-w64.rb

```diff
@@ -33,6 +33,14 @@ class MingwW64 < Formula
     url "https://ftp.gnu.org/gnu/binutils/binutils-2.40.tar.xz"
     mirror "https://ftpmirror.gnu.org/binutils/binutils-2.40.tar.xz"
     sha256 "0f8a4c272d7f17f369ded10a4aca28b8e304828e95526da482b0ccc4dfc9d8e1"
+
+    # Fix linking failures in ld against import lib. Fixed upstream but still
+    # present in 2.40. Can remove this once 2.41 is released.
+    # https://sourceware.org/bugzilla/show_bug.cgi?id=30079
+    #
+    # Patch is from:
+    # https://sourceware.org/git/?p=binutils-gdb.git;a=patch;h=b7eab2a9d4f4e92692daf14b09fc95ca11b72e30
+    patch :DATA
```

Comment:
Thanks for the PR @JoelOtter. It seems the patch can be directly applied, so please do this instead *and remove the embedded patch at the end of the formula*:

```suggestion
    patch do
      url "https://sourceware.org/git/?p=binutils-gdb.git;a=patch;h=b7eab2a9d4f4e92692daf14b09fc95ca11b72e30"
      sha256 "5d22a0be9f35738b3dd7d95dea9f1b3f56f25fe9c92f5e216fb31f1c84325e25"
    end
```

Reply:
We should probably use the raw diff to avoid embedding the Git version they use (this changes sometimes): https://sourceware.org/git/?p=binutils-gdb.git;a=commitdiff_plain;h=b7eab2a9d4f4e92692daf14b09fc95ca11b72e30;hp=0d2f72332c7606fa3181b54dceef82d1af403624

Reply:
Will do! Out of interest, how did you find the `sha256` hash for this patch? I couldn't work it out

Reply:
Will do! Out of interest, how did you find the sha256 hash for this patch? I couldn't work it out.

Reply:
```
curl -L <url> | shasum -a 256
```